scripts/build_binary.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Normalize to working directory being build root (up one level from ./scripts)
+ROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )
+cd "${ROOT}"
+
+go run ./internal/pkg/version/gen/generate-version.go
+
+GIT_SHORT_HASH=`git rev-parse --short=7 HEAD`
+TAGGED=`git tag --points-at ${GIT_SHORT_HASH}`
+
+if [ -z "$TAGGED" ]; then
+  GIT_DIRTY=`echo '*'`
+fi
+
+GIT_HASH="$GIT_DIRTY$GIT_SHORT_HASH"
+
+echo "Building archer to ${DESTINATION}"
+
+# TODO: Inject version and git short hash into build
+GOOS=$GOOS GOARCH=$GOARCH CGO_ENABLED=0 go build -ldflags \
```

Comment:
I think you either want `CGO_ENABLED=$CGO_ENABLED` or you don't need to prefix `CGO_ENABLED=0` on the `build_binary.sh` script invocations in `Makefile`.